tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1058,7 +1074,7 @@ def test_post_notification_without_document_email_verification_permission(
 
     assert (
         resp['errors'][0]['message'] ==
-        'Email verification flow for document download has not been enabled for this service.'
+        "Email verification and/or custom retention for 'send files by email' has not been enabled for this service."
```

Comment:
Might be worth checking this content with Karl?

Reply:
It's only technically an error that a user could ever see by trying to use the feature before we announce/release it.

It's not real content per se - will be removed when we release.

Reply:
Makes sense 👍🏼